invenio_app_ils/records/resolver/loan.py

```diff
@@ -58,7 +58,7 @@ def document_resolver(loan_pid):
         # add only some fields
         obj = {}
         include_document_keys = ["title", "pid",
-                                 "circulation", "authors"]
+                                 "circulation", "authors", "document_type"]
```

Comment:
Did you use this in a facet? Do you expect a `keyword` field?
What about if you use the new [pick](https://github.com/inveniosoftware/invenio-app-ils/blob/master/invenio_app_ils/jsonresolver/api.py#L36) like [this](https://github.com/inveniosoftware/invenio-app-ils/blob/master/invenio_app_ils/acquisition/jsonresolvers/order_vendor.py#L33)?
If you have the courage (maybe next PR), it would be nice to change also the resolvers above `item_resolver` and `loan_patron_resolver` to pick the fields. (`loan_patron_resolver` is more problematic because in CDS we will have more fields, maybe we have to re-define that loader in CDS)